composite/proof-of-reserves/src/adapter.ts

```diff
@@ -1,56 +1,42 @@
 import { logger } from '@chainlink/external-adapter'
-import { AdapterRequest, AdapterResponse, Execute } from '@chainlink/types'
-import reduceAdapter from '@chainlink/reduce-adapter'
-import { getImpl as getProtocolImpl, getProtocol } from './protocol'
-import { getImpl as getBalanceImpl, getBitcoinIndexer } from './balance'
+import { AdapterRequest, ExecuteWithConfig, Config, ExecuteFactory } from '@chainlink/types'
+import { Validator, Requester } from '@chainlink/external-adapter'
+import { makeConfig, makeOptions, getURL } from './config'
+import { runProtocolAdapter } from './protocol'
+import { runBalanceAdapter } from './balance'
+import { runReduceAdapter } from './reduce'
+
+export const makeRequestFactory = (config: Config, prefix: string) => async (
+  input: AdapterRequest,
+) => Requester.request({ ...config.api, method: 'post', url: getURL(prefix, true), data: input })
 
 // Run, log, throw on error
-const runAdapter = async (execute: Execute, input: AdapterRequest, tag: string) => {
+export const callAdapter = async (execute: any, input: AdapterRequest, tag: string) => {
```

Comment:
I might be missing it, but why are we changing the type of `execute` from Execute to any?

Reply:
Shouldn't be! Changed.